tests/amd64/ubuntu_20_04_metadata_failure_test.yaml

```diff
@@ -0,0 +1,12 @@
+schemaVersion: '2.0.0' # Make sure to test the latest schema version
+metadataTest:
+  env:
+  - key: 'SOME_KEY'
+    value: 'SOME_VAL'
+  envVars:
+    - key: 'OTHER_KEY'
+      value: 'SOME_VAL'
+  unboundEnv:
+  - key: 'BAR_FOO'
+  unboundEnvVars:
+  - key: 'BAZ_FOO'
```

Comment:
Based on my other comment, this can be a valid cst test with de-duplication & merging between the two sets of env vars.